Dockerfile

```diff
@@ -7,10 +7,46 @@ ADD . /go-ethereum
 RUN cd /go-ethereum && make geth
 
 # Pull Geth into a second stage deploy alpine container
-FROM alpine:latest
+FROM alpine:3.16.0
+
+ARG BSC_VERSION=v1.1.11
+ARG BSC_USER=bsc
+ARG BSC_USER_UID=1000
+ARG BSC_USER_GID=1000
+
+ENV BSC_HOME=/bsc
+ENV HOME=${BSC_HOME}
+ENV DATA_DIR=/data
+
+RUN apk add --no-cache ca-certificates~=20211220 jq~=1.6 bash~=5.1.16-r2 bind-tools~=9.16.29-r0 tini~=0.19.0 curl==7.83.1-r2 sed~=4.8-r0 \
+  && rm -rf /var/cache/apk/* \
+  && addgroup -g ${BSC_USER_GID} ${BSC_USER} \
+  && adduser -u ${BSC_USER_UID} -G ${BSC_USER} --shell /sbin/nologin --no-create-home -D ${BSC_USER} \
+  && addgroup ${BSC_USER} tty \
+  && sed -i -e "s/bin\/sh/bin\/bash/" /etc/passwd  
+
+RUN echo "[ ! -z \"\$TERM\" -a -r /etc/motd ] && cat /etc/motd" >> /etc/bash/bashrc \
+    && echo "Version: ${BSC_VERSION}" > /etc/motd
+
+WORKDIR ${BSC_HOME}
 
-RUN apk add --no-cache ca-certificates curl jq tini
 COPY --from=builder /go-ethereum/build/bin/geth /usr/local/bin/
 
+COPY docker-entrypoint.sh ./
+
+RUN curl -LO https://github.com/bnb-chain/bsc/releases/download/${BSC_VERSION}/mainnet.zip \
```

Comment:
the URL can be replaced by `.github/release.env`. 
the mainnet and testnet config of the new release is obtained here

Reply:
`release.env` still pointing to `v1.1.8` and that is why I have a ARG to build on.

Reply:
Actually every time a new version is released the additional files come from here, so it will be the same if `releas.env` doesn't update

The reason I want you to use `release.env` is that developers will most likely forget to update the Dockerfile when they release a new version

Reply:
when each release, no need to update the `BSC_VERSION` in `Dockerfile`, the build process should pass in `--build-arg` for the new version to be used, similar to the new target `docker` in makefile. Additionally, this won't cause confusion for  developer between the regular release in the github`- repo and configuration version being used in Dockerfile.